setup.cfg

```diff
@@ -28,7 +28,7 @@ license = Apache 2.0
 license_file = LICENSE
 
 [options]
-packages = find:
+packages = find_namespace:
```

Comment:
The second one does the same as the former but just adds the feature to also add namespace packages (apart from regular ones).